source/Octo/Commands/PackCommand.cs

```diff
@@ -127,9 +129,10 @@ public Task Execute(string[] commandLineArguments)
                 if (!string.IsNullOrWhiteSpace(title))
                     metadata.Title = title;
 
-                log.Information("Packing {PackageId:l} version {Version}...", id, version);
+                log.Information($"{(verbose ? "Verbose logging" : "")}");
```

Comment:
Yes that's better, not sure why I decided to log an empty line there (I may have had it saying 'non verbose' while testing)...